components/admin_console/cloud_migrate/migrate_workspace.tsx

```diff
@@ -0,0 +1,255 @@
+    const [toBackButton] = useState('null');
+    const [percentage, setPercentage] = useState(1);
+
+    // const [step, setStep] = useState(0);
+
+    useInterval(() => {
+        setPercentage(percentage + 1);
+    }, 1000);
+
+    useEffect(() => {
+    }, []);
+
+    const adminHeader = () => (
+        <AdminHeader className={`admin-console__migrate-header ${toBackButton ? 'with-back' : ''}`}>
+            <>
+                {toBackButton &&
+                <BlockableLink
+                    to='/admin_console/billing/payment_info'
+                    className='fa fa-angle-left back'
+                />}
+                <FormattedMessage
+                    id='admin.cloud.migrate.header.title'
+                    defaultMessage='Import a Workspace'
+                />
+                <Badge
+                    className='BetaBadge'
+                    show={true}
+                >
+                    <FormattedMessage
+                        id='admin.cloud.migrate.header.BetaBadge'
+                        defaultMessage='Beta'
+                    />
+                </Badge>
+                {/* this probably should go to the shared admin header and the state to the main redux store*/}
+                {shouldShowImportInProgressHeaderChip() && <ProgressBar percentage={percentage}/>}
+            </>
+        </AdminHeader>
+    );
+
+    const shouldShowImportInProgressHeaderChip = () => {
+        return true;
+    };
+
+    const shouldShowImportInProgressAlertBanner = () => {
+        return true;
+    };
+
+    const shouldShowImportRecommended4NewWorkspacesAlertBanner = () => {
+        return true;
+    };
+
+    const importInProgress = (
+        <AlertBanner
+            mode='info'
+            title={
+                <FormattedMessage
+                    id='admin.cloud.migrate.importInProgress.banner.title'
+                    defaultMessage='Import in progress'
+                />
+            }
+            message={
+                <>
+                    <FormattedMarkdownMessage
+                        id='admin.cloud.migrate.importInProgress.banner.description'
+                        defaultMessage='New imports are disabled until this import is complete. You may continue using Mattermost while the import progresses and we will email you once the import is complete.'
+                    />
+                    <a
+                        target='_new'
+                        rel='noopener noreferrer'
+                        href={CloudLinks.BILLING_DOCS}
+                        className='migrationInProgress'
+                    >
+                        <FormattedMessage
+                            id='admin.cloud.migrate.importInProgress.banner.link'
+                            defaultMessage='View in-progress import'
+                        />
+                    </a>
+                </>
+            }
+        />
+    );
+
+    const importRecommended4NewWorkspaces = (
```

Comment:
```suggestion
    const importRecommendedForNewWorkspaces = (
```

Reply:
Could you spell this out in all places? There are still some translation ids that have `importRecommended4NewWorkspaces`